### PR TITLE
Increase zstd frame size for compressed access log archive

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by freva — APPROVED** [review]

File: jdisc_http_service/src/main/java/com/yahoo/container/logging/LogFileHandler.java [hunk 1]
```diff
@@ -401,7 +401,7 @@ private static void runCompressionNone(NativeIO nativeIO, Path oldFile) {
         private static void runCompressionZstd(NativeIO nativeIO, Path oldFile) {
             try {
                 Path compressedFile = Paths.get(oldFile.toString() + ".zst");
-                int bufferSize = 0x400000; // 4M
+                int bufferSize = 48*1024*1024;
```
Comment (baldersheim) [inline jdisc_http_service/src/main/java/com/yahoo/container/logging/LogFileHandler.java:404]: I think this is on the high side. This means that you will have a 48M buffer on even the smallest container.
Is this really necessary ? What is the gain. 4M sound like plenty.

Comment (bjorncs) [reply jdisc_http_service/src/main/java/com/yahoo/container/logging/LogFileHandler.java:404]: The motivation was to improve compression ratio compared gzip. The zstandard frames are independent, so bigger frame should reduce the overall file size (for large access logs).

Comment (bjorncs) [reply jdisc_http_service/src/main/java/com/yahoo/container/logging/LogFileHandler.java:404]: You could change it to config and use smaller value for non-application container cluster.

Comment (baldersheim) [reply jdisc_http_service/src/main/java/com/yahoo/container/logging/LogFileHandler.java:404]: Have you measured the gain. When I tested earlier on there was not very much gain after 1M. Then it is better to increase compression level.

**Pull request merged**